aptos-move/framework/aptos-framework/sources/multisig_account.move

```diff
@@ -633,6 +647,57 @@ module aptos_framework::multisig_account {
         update_metadata_internal(multisig_account, keys, values, true);
     }
 
+    /// Allow the owners of a multisig to remove multiple pending transactions from the queue in a single transaction.
+    /// This requires sufficient signed messages from the owners to meet the signature threshold.
+    /// This also effetively allows the batch removal to be done out-of-order with other pending transactions in the
+    /// queue.
+    entry fun batch_remove_transactions(
+        multisig_account: address,
+        owners_signing: vector<address>,
+        owner_accounts_schemes: vector<u8>,
+        owner_public_keys: vector<vector<u8>>,
```

Comment:
This appears to be returning to some of the patterns of the multi-ed25519 paradigm